test/external-dependencies/boost.cpp

```diff
@@ -22,12 +22,14 @@
 
 #include <boost/test/unit_test.hpp>
 #include <libdevcore/Common.h>
+#include <test/TestHelper.h>
 
 BOOST_AUTO_TEST_SUITE(ExtDepBoost)
 
 // test that reproduces issue https://github.com/ethereum/cpp-ethereum/issues/1977
 BOOST_AUTO_TEST_CASE(u256_overflow_test)
 {
+	dev::test::TestOutputHelper::initTest();
```

Comment:
I see that this is called in every test. Isn't there some functionality/option in the boost test library that you can use?
I guess they have test reporting tool integrated and you don't need to build your own, rather use theirs.